src/plus/integrations/providers/github/github.ts

```diff
@@ -455,6 +457,7 @@ export class GitHubApi implements Disposable {
 
 			return {
 				provider: provider,
+				id: author.login ?? '',
```

Comment:
Well, in the case above I've introduced a concept of `UnidentifiedAuthor` = an author without an account, who therefore doesn't have an `id`. That is reasonable because when you ask for an account for _commit_, there _can be_ an author without an account. So, I let the result be an object without a `login/id`.

This case is different. Here we ask for _an account for email,_ therefore here either we have an existing account that has to have a `login`, or the account does not exists, so we should get `null` result.

Do you agree?